pkg/apis/eksctl.io/v1alpha5/vpc.go

```diff
@@ -55,8 +55,8 @@ type (
 
 	// ClusterEndpoints holds cluster api server endpoint access information
 	ClusterEndpoints struct {
-		PrivateAccess *bool `json:"privateAccess,omitempty,false"`
-		PublicAccess  *bool `json:"publicAccess,omitempty,true"`
+		PrivateAccess *bool `json:"privateAccess,omitempty"`
```

Comment:
false or true is invalid `json` tag. The default values are set in defaults.go `ClusterEndpointAccessDefaults`